### DISCUSSION TITLE
Favoritenliste wird genausooft angezeigt wie man User dafür verbindet

### DISCUSSION THREAD

**Discussion by grmpf72** [OpenCCU/OpenCCU#2685] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Unter Einstellungen/Favoriten wird die Favoritenliste (habe nur eine) genausooft angezeigt wie man Benutzer damit verbindet!

### Describe the behavior you expected

Favoritenliste wird nur einmal angezeigt - egal wie viele Benutzer damit verbunden sind.

### Steps to reproduce the issue

In Einstellungen/Favoriten die Favoriten bearbeiten, mehrere User damit verbinden und dann speichern.

### What is the version this bug report is based on?

 	3.75.6.20240316

### Which base platform are you running?

rpi3 (RaspberryPi3, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
no, sorry
```

**Comment by jens-maus** [comment]
Bitte mit Screenshot zeigen was du meinst.

**Comment by grmpf72** [comment]
<img width="1144" alt="Screenshot 2024-03-18 105401" src="https://github.com/jens-maus/RaspberryMatic/assets/163859179/1c57411f-b11b-4bda-9d7f-754f4c4c2c0c">
Alle 5 vorhandenen Benutzer mit der Favoritenliste verbunden = 5x Anzeige der Liste

**Comment by jens-maus** [comment]
Du unterschlägst in deinem Screenshot leider die wichtige Spalte "Benutzer" ganz rechts.

**Comment by grmpf72** [comment]
![Screenshot 2024-03-18 141537](https://github.com/jens-maus/RaspberryMatic/assets/163859179/a7d9b35d-bbee-4b98-924d-b851d88074bc)
Sorry - du meintest das hier?
Wenn man einen Benutzer anhakt kommt die Liste nur einmal. Ansonsten ein Duplikat für jeden Angehakten. Wenn man bei einem LÖSCHEN auswählt ist die Liste aber für alle weg.

**Comment by jens-maus** [comment]
Nein, ich meinte exakt in dem 1. screenshot den du da gezeigt hast fehlt recht doch noch zwei spalten... du hast hier nur einen teil der ansicht gezeigt.

**Comment by jens-maus** [comment]
na sieht doch gut und normal aus. jeder der 5 benutzer hat hier natürlich seine eigene Ansicht damit man dort in den Ansichten auch die Reihenfolge nutzerbasiert machen kann. Das ist kein Bug, sondern schlichtweg ein Missverständnis deinerseits.

**Comment by grmpf72** [comment]
Sorry, aber da steht bei Reihenfolge überall 1. Das kann nicht korrekt sein. Ich habe gerade nochmal auf der Original-CCU geguckt: Da ist es genau einmal zu sehen wenn 5 Nutzer angehakt sind. Außerdem woher soll man wissen, welche der 5 Favoriten für welchen User ist. Nene! Da stimmt was gar nicht.
Ist doch auch ganz einfach zu testen: Leg mal Favoriten an, verbinde die mit mehr als einem Benutzer und dann guck mal was passiert.

**Reply by jens-maus** [reply]
Gut, es ist aber definitiv kein Bug/Problem. Aber gut, dann verschiebe ich das eben in den Diskussionsbereich.

**Reply by jens-maus** [reply, marked as answer]
Du verstehst die Nutzung hier einfach nur falsch. Füge mal bei einem Nutzer einen zweiten Favoriten hinzu dann wird es vielleicht klarer. Wenn du nämlich in der Lage sein willst die Favoriten zu sortieren (d.h. die Reihenfolge zu setzen) dann gelingt dir das nur in RaspberryMatic und nicht in der "Original-CCU" Firmware.

**Reply by grmpf72** [reply]
OK - habe das mal auf meinem Test-Raspi mit RaspberryMatic nachgestellt. Ist wohl tatsächlich ein Feature - dessen Sinn sich MIR NOCH nicht erschließt. Wenn ich mal ne Stunde Zeit dafür habe spiele ich mal damit rum. Stören tuts (mich) auf jeden Fall nicht. Danke!
